[testing travis] Send to contact form (#163)
CodeYourFuture/syllabus: docs/js-core-1/week-1/lesson.md

1. Write a function that returns the year someone is born given their age as input
2. Using the answer from step 1, write a function that takes someone's name and age as input and returns a string that states the person's name and year they were born in a sentence

## Giving Feedback to Each Other

A critical part of being developer is reading other people's code and giving good feedback and improvements. This can happen even when you don't know a lot of code - everybody has unique ideas and insights into how to make code better!

Let's spend the next five minutes reviewing CodeYourFuture's Style Guide.

> This style guide is how we expect you to write code whilst you're at CodeYourFuture - each company has one and it's really important you stick to it to make sure we all write code that other people can understand.

Reviewer: ```suggestion
This style guide is how we expect you to write code whilst you're at CodeYourFuture - most companies have one and it's really important you stick to it to make sure we all write code that other people can understand.
```
